app/components/administrative-body-select.hbs

```diff
@@ -0,0 +1,18 @@
+<div class={{if @error "ember-power-select--error"}}>
```

Comment:
Not sure if that's the proper way to add an error state to PowerSelect. I couldn't find it in the docs and looked in the styles instead. It looks a bit weird since it's a BEM modifier though but the `ember-power-select` class doesn't seem to include styles. So I would only add it to follow the BEM rules. Renaming the class might make more sense though.

Reply:
tbh I have no idea, perhaps @Dietr  has some pointers